RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -94,12 +95,13 @@ public void setGoalRotation(double degrees)
 
     public double getCurrentRotation()
     {
-        return INTAKE_RAISER_MOTOR_LEFT.getEncoder().getPosition();
+        return INTAKE_RAISER_MOTOR_RIGHT.getEncoder().getPosition();
```

Comment:
Change this back to the left.

Reply:
This should be RIGHT because LEFT is supposed to follow RIGHT.